File: Leet-Code/1342-Number-of-Steps-to-Reduce-a-Number-to-Zero/NumberOfSteps.cs
public class Solution {
    public int NumberOfSteps(int num) {

        var stepCount = 0;
        while(num != 0)
        {
            num = num % 2 == 0 ? num / 2 : num - 1;
            stepCount++;
        }
        
        return stepCount;
    }
}

File: Leet-Code/1342-Number-of-Steps-to-Reduce-a-Number-to-Zero/NumberOfSteps.py
class Solution:
    def numberOfSteps(self, num: int) -> int:
        stepCount = 0
        while(num != 0):
            if(num % 2 == 0):
                num = num / 2
                stepCount += 1
            else:
                num = num - 1
                stepCount += 1
                 
        return stepCount
